lib/ryan.rb

```diff
@@ -18,9 +18,19 @@ class Ryan
   def_delegators :const,
     :name, :funcs, :type, :initialization_args, :func_by_name, :class?, :module?
 
+  def self.root
+    root_path = Pathname.new File.expand_path('../..', __FILE__)
+
+    return root_path
```

Comment:
I prefer to end all methods with a return line. Being explicit about what the method returns saves headache down the road.